app/lib/main.dart

```diff
@@ -18,12 +20,26 @@ void main() {
 class OdsApplication extends StatelessWidget {
   const OdsApplication({super.key});
 
+  static List<Recipe> recipes = [];
+  static List<Category> category = [];
+  static List<Food> foods = [];
+
+  // Fetch content from the json file
+  Future<void> readJson() async {
+    final String response = await rootBundle.loadString('assets/recipes.json');
+    Entity entity = entityFromJson(response);
+
+    category = entity.category;
+    recipes = entity.recipes;
+    foods = entity.foods;
+  }
+
   @override
   Widget build(BuildContext context) {
+    readJson();
```

Comment:
I'm not sure this is where it should be. As the `build` documentation says:
```
  /// Describes the part of the user interface represented by this widget.
  ///
  /// The framework calls this method when this widget is inserted into the tree
  /// in a given [BuildContext] and when the dependencies of this widget change
  /// (e.g., an [InheritedWidget] referenced by this widget changes). This
  /// method can potentially be called in every frame and should not have any side
  /// effects beyond building a widget.
  ```

So I think this should not be called there but as I'm not a Flutter expert I don't know where it would be better...